altos-rust/altos-core/libs/heap/free_list_allocator/src/lib.rs

```diff
@@ -65,9 +65,9 @@ pub extern fn __rust_allocate(size: usize, align: usize) -> *mut u8 {
 
 #[no_mangle]
 #[cfg(not(test))]
-pub extern fn __rust_deallocate(_ptr: *mut u8, _size: usize, _align: usize) {
+pub extern fn __rust_deallocate(ptr: *mut u8, size: usize, align: usize) {
```

Comment:
The underscore in front of a variable name marks it as unused, so we don't get compiler warnings for unused variables. The doc examples show them as wrappers around a C style malloc library that only takes the pointer to the memory to be freed, and doesn't use the `size` or `align` parameters. In our case, however, we are using those parameters so they shouldn't have an underscore in front of them.